demux/demux_mkv.c

```diff
@@ -1522,10 +1522,8 @@ static int demux_mkv_open_video(demuxer_t *demuxer, mkv_track_t *track)
 
     sh_v->crop.x0 = track->v_crop_left_set ? track->v_crop_left : 0;
     sh_v->crop.y0 = track->v_crop_top_set ? track->v_crop_top : 0;
-    sh_v->crop.x1 = track->v_width -
-                        (track->v_crop_right_set ? track->v_crop_right : 0);
-    sh_v->crop.y1 = track->v_height -
-                        (track->v_crop_bottom_set ? track->v_crop_bottom : 0);
+    sh_v->crop.x1 = track->v_crop_right_set ? track->v_width - track->v_crop_right : 0;
+    sh_v->crop.y1 = track->v_crop_bottom_set ? track->v_height - track->v_crop_bottom : 0;
```

Comment:
This doesn't work, because if you have let's say only `v_crop_right_set == true`, you will end up with crop Wx0+0+0 which is invalid.

Reply:
Wouldn't the same thing happen before? I thought detecting a file with broken crop parameters was a feature.

Reply:
Oh nevermind I get what you mean now.